cmd/soci/commands/index/rm.go

```diff
@@ -47,7 +47,7 @@ var rmCommand = cli.Command{
 		}
 
 		ctx := context.Background()
-		ctx, contentStore, err := store.NewContentStore(ctx, internal.ContentStoreOptions(cliContext)...)
+		contentStore, err := store.NewContentStore(internal.ContentStoreOptions(cliContext)...)
```

Comment:
Before this change, `store.NewContentStore` would set the namespace on the context. If we're not updating the context anymore, we need to set that ourselves or use the context from `internal.NewClient`

This probably applies to some of the other CLI commands as well, so can you double check them all?

Reply:
Did a walk through and made some changes for this change. In the process I found that we don't actually respect timeouts in a few commands, so I made sure every command honors the `--timeout` flag.